### PR TITLE
AW-127 : Made slider loop and added animation

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/slider/Slider.java [hunk 1]
```diff
@@ -97,6 +100,16 @@ public void onAttachOrDetach(AttachEvent event) {
         if (event.isAttached()) {
             index = 0;
             displayCurrent();
+            $(contents).on(TRANSITION_END, new Function() {
+                @Override
+                public void f() {
+                    GQuery.console.log("TRANSITION_END");
```
Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/slider/Slider.java]: You should remove console outputs for production

**Comment by jasonlemay** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/slider/Slider.java (hunk 1 above)
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/slider/Slider.java]: Good catch!

**Pull request merged**